array-methods/filter.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evens = numbers.filter((element) => element % 2 === 0);
+console.log(evens);
+const minusDd = names.filter(
+  (element) =>
+    element.includes('d') === false && element.includes('D') === false
```

Comment:
It's more succinct to say `!element.includes('d')` instead of `element.includes('d') === false`.
You will only very rarely ever need to compare something directly to `true` or `false`.